env/index.ts

```diff
@@ -0,0 +1,78 @@
+export type Env = {
```

Comment:
Does it make sense to bring in `ARCJET_KEY` here as well?

Reply:
Yep. I was planning to add a helper function for it when we added cloudflare, but I can just set that up now.